stylesheet.css

```diff
@@ -0,0 +1,310 @@
+  display: grid;
+  grid-template-columns: 1fr min-content;
+}
+
+.tweet .postOptions {
+  font-size: small;
+  margin-top: 20px;
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(auto-fill, minmax(45px, 1fr));
+}
+
+.tweet .postOptions img {
+  height: 20px;
+}
+
+.OptionsAndSave .postType {
+  display: grid;
+  grid-template-columns: repeat(6, min-content);
+  grid-gap: 3px;
+  align-items: start;
+  justify-items: center;
+}
+
+.postType img {
+  height: 20px;
+  border-radius: 100%;
+  padding: 10%;
+  margin: 0 5px;
+}
+
+.postType img:hover {
+  background-color: rgb(255, 206, 226);
+}
+
+.tweet {
+  background-color: rgba(255, 255, 255, 0.8);
+  border-radius: 10px;
+  padding: 20px;
+  margin: 20px;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: 50px 1fr;
+  box-shadow: 5px 5px 10px rgba(98, 27, 85, 0.1);
+}
+
+.tweet:hover {
+  cursor: pointer;
+
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.profileName + .handle, .handle + .timestamp {
+  margin-left: 10px;
+}
+
+.tweet .timestamp {
+  font-style: italic;
+  color: rgba(98, 27, 85, 0.5);
+  font-size: smaller;
+}
+
+.tweet .handle {
+  color:rgba(236, 72, 135, 0.5);
+}
+
+.tweet .text {
+  margin-top: 1em;
+  line-height: 150%;
+}
+
+
+.tag, .mention, .profileName, .news p, .followSuggestions p {
+  color: rgb(236, 72, 135);
+}
+
+.profileName {
+  font-weight: bold;
+}
+
+.tag:hover, .mention:hover, .handle:hover {
+  color: rgb(132, 30, 69);
+}
+
+.timestamp:hover {
+  text-decoration: underline;
+}
+
+.tag:before, .news p::before {
+  content: "#"
+}
+
+.mention:before, .handle:before, .followSuggestions p::before {
+  content: "@";
+}
+
+.main {
+  grid-area: "main";
+  max-height: 100vh;
+  overflow-x: none;
+  overflow-y: scroll;
+}
+
+.sidebar {
+  grid-area: "sidebar";
+}
+
+.searchBar, .news, .followSuggestions {
+  border-radius: 10px;
+  padding: 20px;
+  margin: 10px;
+  border: 1px solid rgb(236, 72, 135, 0.5);
+  background-color: rgba(255, 255, 255, 0.8);
+  overflow: hidden;
+}
+
+.news p, .followSuggestions p {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  margin: 10px 0;
+}
+
+.searchBar {
+  padding: 10px;
+  color: rgb(236, 72, 135, 0.5);
+  font-style: italic;
+  border-radius: 20px;
+}
+
+h1,
+h2 {
+  font-size: 1em;
+  margin: 0;
+}
+
+#logo {
+  padding: 10px;
+}
+
+.verified::after {
+  content: url(assets/verified.svg);
+  height: 20px;
+}
+
+.profileImg {
+  height: 50px;
+  width: 50px;
+  border-radius: 100%;
+  overflow: hidden;
+}
+
+.profileImg > img {
+  height: 50px;
+  object-fit: fill;
+}
+
+.embeddedMedia {
+  margin: 10px 0;
+  background-color: black;
+  border-radius: 5px;
+  object-fit: fill;
+  overflow: hidden;
+  display: grid;
+  max-height: 400px;
+  place-content: center;
+  place-items: center;
+}
+
+.embeddedMedia > img {
+  width: 90%;
+  object-fit: fill;
+}
+
+.icon {
+  height: 16px;
+}
+
+.icon path {
+  fill: rgb(236, 72, 135, 0.5);
+}
+
+.rainbow {
```

Comment:
Is this where you got your beautiful background colour gradient?

Reply:
Yeah haha! If you don't feel like manually doing all that, https://cssgradient.io/ does all the heavy lifting for you and just gives you the background-param config